test/run

```diff
@@ -21,6 +21,7 @@ RE_EXPENSIVE='E2E'
 
 # every known case needs to set RUN_OPTS to something non-empty, so that we can check if we hit any branch
 case "${TEST_SCENARIO:=}" in
+    *efi*) RUN_OPTS="$ALL_TESTS"; export TEST_FIRMWARE="efi" ;;&
```

Comment:
I am confused by the '&' at the end of the line.

Reply:
It will make the `case` evaluation continue with the next one. I.e. the scenarios are not mutually exclusive, but combine.

Reply:
he ;& in a case statement in a Bash script allows execution to continue with the next pattern block in the case statement, regardless of whether the next pattern matches or not. 

The 'efi' test scenario it not mutually exclusive with the `expensive` and `other` scenarios, they can be used together.
So `expensive/efi`, `other/efi`. I will update the documentation in test/README.rst to reflect that.